gcloud-java-core/src/main/java/com/google/gcloud/Page.java

```diff
@@ -14,12 +14,12 @@
  * limitations under the License.
  */
 
-package com.google.gcloud.storage;
+package com.google.gcloud;
 
 /**
- * Interface for Google Cloud storage list result.
+ * Interface for Google Cloud paginated results.
  */
```

Comment:
Shall we include a code snippet to suggestion how Page based results should be consumed?